packages/access-api/src/utils/email.js

```diff
@@ -32,6 +32,7 @@ export class Email {
           product_name: 'Web3 Storage',
           email: opts.to,
           action_url: opts.url,
+          match_phrase: opts.nonce,
```

Comment:
I think it's fine to have `match_phrase` in snake case here but in all other places we should be using camelCase to be consistent...